File: src/MSAmanager/app/src/main/java/ms_alum/azrieli_college_of_engineering/com/msamanager/project_options.java
package ms_alum.azrieli_college_of_engineering.com.msamanager;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class project_options extends AppCompatActivity {

    private Button btn_kablni_mishne;
    private Button btn_divohi_takalot;
    private Button btn_loz;
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_project_options);
        //subroutine to connect graphical objects to their programical representatoins

        btn_divohi_takalot = (Button)findViewById(R.id.project_options_btn_divohi_takalot);
        btn_kablni_mishne = (Button)findViewById(R.id.project_options_btn_kablni_mishne);
        btn_loz = (Button)findViewById(R.id.project_options_btn_loz);


        btn_kablni_mishne.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(project_options.this,kablni_mishne_activity.class);
                startActivity(intent);
            }
        });


        btn_divohi_takalot.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(project_options.this,divohi_takalot_activity.class);
                startActivity(intent);
            }
        });

        btn_loz.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(project_options.this,loz_activity.class);
                startActivity(intent);
            }
        });
    }
}

File: src/app/src/main/java/com/example/chaosruler/msa_manager/activies/project_options.kt
package com.example.chaosruler.msa_manager.activies

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.widget.Button
import com.example.chaosruler.msa_manager.BuildConfig
import com.example.chaosruler.msa_manager.R
import com.example.chaosruler.msa_manager.activies.loz_activity.loz_activity
import com.example.chaosruler.msa_manager.activies.testing_do_all_table_activities.table_chooser
import com.example.chaosruler.msa_manager.services.global_variables_dataclass
import com.example.chaosruler.msa_manager.services.themer
import kotlinx.android.synthetic.main.activity_project_options.*

/**
 * Class is responsible for the logic behind the project options activity (choosing what to do inside a project)
 * @author Chaosruler972
 * @constructor as an activity, constructor is default constructor
 */
class project_options : Activity() {

    /**
     * define the project name we are working on, also refreshes the global variables to know that we chose
     * that project (by name), on case that we didn't actually choose any project, activity automaticily closes
     * function is part of the activity lifecycle of android activities
     * @author Chaosruler972
     * @param savedInstanceState the state that we opened the activity last time
     */
    override fun onCreate(savedInstanceState: Bundle?) {
        setTheme(themer.style(baseContext))
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_project_options)
        if(!global_variables_dataclass.GUI_MODE)
            global_variables_dataclass.projid = intent.getStringExtra(getString(R.string.key_pass_main_to_options))
        else
            global_variables_dataclass.projid = ""
        if(global_variables_dataclass.projid == null)
            finish()
        init_buttons()

    }

    /**
     * inits buttons
     * @author Chaosruler972
     */
    private fun init_buttons()
    {

        project_options_btn_divohi_takalot.setOnClickListener(
        {
            val intent = Intent(this@project_options, divohi_takalot_activity::class.java)
                    startActivity(intent)
        })

        project_options_btn_kablni_mishne.setOnClickListener(
        {
            val intent = Intent(this@project_options, kablni_mishne_activity::class.java)
                    startActivity(intent)
        })

        project_options_btn_loz.setOnClickListener(
        {
            val intent = Intent(this@project_options, loz_activity::class.java)
                    startActivity(intent)
        })


    }

    




}
